src/main/js/controls/Line/Line.js

```diff
@@ -158,23 +169,33 @@ class Line extends GraphContent {
      * @inheritdoc
      */
     resize(graph) {
-        if (utils.notEmpty(this.dataTarget.regions)) {
+        if (
+            utils.notEmpty(this.dataTarget.regions) ||
+            !utils.isEmptyArray(this.dataTarget.valueRegionSubset)
```

Comment:
Since we have notEmpty method in utils. Can we use it in here instead of !isEmptyArray

Reply:
valueRegionSubset is an array. utils.notEmpty checks only object or string. So I have used utils.isEmptyArray.